### ISSUE TITLE
systemctl start keylime_registrar hangs (waiting for password or timeout?)

### ISSUE THREAD

**Issue by jronnblom** [keylime/keylime-deb#5]
If you check with ps when start hangs you find this:
```

\_ systemctl start keylime_registrar
    \_ /bin/systemd-tty-ask-password-agent --watch

```

**Comment by galmasi** [comment]
Good point. We have only been testing the keylime agent in this packaging. I wonder whether we should remove packaging for registrar and verifier services, since no one is testing. Are you testing debian-packaged keylime verifier/registrar?

**Comment by jronnblom** [comment]
Yes. On and off. Trying to get it installed but on Ubuntu 18.04.

**Comment by jronnblom** [comment]
Actually it doesnt hang. It just stays there and then after a minute or so it starts.
